applications/luci-app-unbound/luasrc/model/cbi/unbound.lua

```diff
@@ -125,6 +131,15 @@ wfq:value("4", translate("Interface FQDN, All Addresses"))
 wfq:depends({ dhcp_link = "none" })
 wfq:depends({ dhcp_link = "odhcpd" })
 
+exa = s1:taboption("advanced", ListValue, "add_extra_dns", translate("Extra DNS:"),
```

Comment:
Shouldn't this extra code for setting add_extra_dns be included in PR#4361 ?
I gather not since this relates to another package (luci-app-unbound, not unbound). Maybe an install-time dependency on the minimal required version of the unbound package is appropriate.